x/evm/keeper/msg_server.go

```diff
@@ -1458,13 +1458,19 @@ func (s msgServer) SignCommands(c context.Context, req *types.SignCommandsReques
 	}
 
 	commandList := types.CommandIDsToStrings(commandBatch.GetCommandIDs())
-	for _, commandID := range commandList {
-		s.Logger(ctx).Debug("signing command batch", "commandBatchID", batchedCommandsIDHex, "commandID", commandID)
+	for _, commandIDHex := range commandList {
+		commandID, err := strconv.ParseUint(commandIDHex, 16, 64)
+		if err != nil {
+			return nil, err
+		}
+
+		s.Logger(ctx).Debug(fmt.Sprintf("signing command batch %s", batchedCommandsIDHex), "commandBatchID", batchedCommandsIDHex, "commandIDHex", commandIDHex, "commandID",
```

Comment:
Haha, related fix: https://github.com/axelarnetwork/axelar-core/pull/1220/files#diff-4a401bcb508aac66739d477b623d017e5d9aa737701e568f8f947775965938d7R1465
I prefer adding the values in the log itself rather than attributes since it's much easier to search datadog with that.

Reply:
I can drop my line change here if you wanna combine that logic

Reply:
My preference at the moment is to have both :D

Reply:
Think we should always at minimum have attributes as the formal method for querying logs. Prefer to have both as logging some info in the message makes quick searches easier.

Need to agree on these points in the issue you made.

Reply:
facets definitely seem like the best way to correlate logs through various stages since it doesn't require the user to add that key value information into the message string itself, but I will say that a downside is if we get into a situation where we are using many different facet names for the same information, forcing us to union many different facets to get a complete picture versus just being able to search for one string value. that being said, we just need to coordinate on naming conventions (how to capitalize, when to use underscores, etc) -- it may be useful to have a sort of internal library that can codify some of these standards

Reply:
👍 👍 👍 could you please add this comment here #1227  @wilcollins ?